<launch>
    <arg name="veh" default="robot"/>
    <!-- 工程五館:  24.577241522567942, 121.2723557919121
            竹湖:  24.788915, 121.000256
    
    -->
    <arg name="latitude" default="24.785999"/>
	<arg name="longitude" default="120.997481"/>
    <arg name="gps_duplicate" default="false"/>
    <arg name="imu_rotate" default='0' />
    <arg name="imu_name" default="imu/data" />
    <arg name="gps_name" default="navsat/fix" />
    <arg name="utm_zone" default='51' />

    <group ns="$(arg veh)">
        <node pkg="localization" type="localization_uwb_gps_imu.py" name="localization_uwb_gps_imu" respawn="false" output="screen">
            <param name="latitude" value="$(arg latitude)"/>
            <param name="longitude" value="$(arg longitude)"/>
            <param name="imu_rotate" value="$(arg imu_rotate)" />
            <param name="namespace" value="$(arg veh)" />
            <param name="imu_name" value="$(arg imu_name)" />
            <param name="gps_name" value="$(arg gps_name)" />
            <param name="utm_zone" value="$(arg utm_zone)" />
            <param name="veh" value="$(arg veh)" />
        </node>
    </group>

</launch>